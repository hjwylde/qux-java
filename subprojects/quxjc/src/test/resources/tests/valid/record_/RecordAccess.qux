package valid.record_

{int x, str y} f({int x, int y} xs):
    xs = {x: 2, y: 'foo'}

    return xs

void main([str] args):
    xs = {x: 0, y: 1}
    print xs.x
    print xs.y
    print f(xs).y
    print xs.y

