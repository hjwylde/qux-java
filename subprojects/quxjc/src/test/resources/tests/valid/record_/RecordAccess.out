0
1
foo
1
